scripts/actions/spells/trust/koru-moru.lua

```diff
@@ -24,7 +24,11 @@ spellObject.onMobSpawn = function(mob)
     mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 50, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.CURE)
 
     mob:addSimpleGambit(ai.t.MELEE, ai.c.NOT_STATUS, xi.effect.HASTE, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.HASTE)
-    mob:addSimpleGambit(ai.t.CASTER, ai.c.NOT_STATUS, xi.effect.REFRESH, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.REFRESH)
+    mob:addGambit(ai.t.CASTER, {
+        { cond = ai.c.NOT_STATUS, arg = xi.effect.REFRESH },
```

Comment:
I think we don't need `cond` and `arg` here, the gambits are designed to somewhat be read aloud - I suppose the table of multiple conditions is just inferring conjunction and different conditions would be different gambits, inferring disjunction.